app/src/main/java/org/wikipedia/analytics/eventplatform/EventPlatformClient.kt

```diff
@@ -111,7 +111,7 @@ object EventPlatformClient {
          */
         private const val WAIT_MS = 30000L
         private const val TOKEN = "sendScheduled"
-        private val MAX_QUEUE_SIZE = if (ReleaseUtil.isDevRelease) 4 else 128
+        private val MAX_QUEUE_SIZE = if (ReleaseUtil.isDevRelease) 1 else 128
```

Comment:
What do you think about making this number into a Preference (defaulted to 128), and exposing it in Developer prefs, so that we could set it to anything we like?

Reply:
Good idea.. and done